eventcheck/epochcheck/epoch_check.go

```diff
@@ -92,19 +98,20 @@ func CheckTxs(txs types.Transactions, rules opera.Rules) error {
 	if rules.Upgrades.Sonic {
 		maxType = 3
 	}
+	limit := gaspricelimits.GetMinimumFeeCapForEventEmitter(baseFee)
```

Comment:
There is a semantic change here, 
The previous code compared the price to a constant (which was already satisfied by the ingress validation) Therefore no tx would be prevented to be fetched by this reason. 
The new code may reject more transactions, and this could have unforeseen consequences of tx not being discarded. 
long story short: The previous code was redundant, the new one adds constraints.

Reply:
The previous was mostly redundant, yes. But `MaxGasPrice` is not a constant. It is a rule parameter that changes over time. This one in particular is altered on the Fantom mainnet after every epoch.

So also in the previous version, transactions that passed the incoming transaction-pool filter might get rejected at this point.